packages/compiler-ssr/src/transforms/ssrInjectFallthroughAttrs.ts

```diff
@@ -11,7 +11,9 @@ import {
 } from '@vue/compiler-dom'
 
 const filterChild = (node: ParentNode) =>
-  node.children.filter(n => n.type !== NodeTypes.COMMENT)
+  node.children.filter(
+    n => n.type !== NodeTypes.COMMENT && n.type !== NodeTypes.TEXT,
```

Comment:
Doesn't this also need to check that the text node only contains whitespace?

e.g. In this playground the attrs are incorrectly inherited in SSR mode because the text node is ignored:

- [Playground](https://deploy-preview-12304--vue-sfc-playground.netlify.app/#__SSR__eNp9UctqwzAQ/JVFFHIJ9qE9BbfQhhxa6IOmR12MvXGUypKQ1q6D8b9XkuvUgZCbdh5idqdnj8YkbYNsxTJXWGEIHFJjHrgStdGW4PW43gtZws7qGhZJ+jcH04KrLB1dXu8HwtrInNBPANnkdHSUeM9ZoaW2K7BYcgap12TpzMCWjFyh1U5UycFp5RP14Zvgq42QaN8NCa0cZyuITOByKfXPS8TINric8GKPxfcF/OC6gHH2YdGhbZGzE0e5rZBGerN9w86/T2Sty0Z69RXyE52WTcg4yp4aVfrYM11M+xzvKlT15TYdoXLTUiFoUA5Rz5m/8PrK6v9xb5O76ONq8FecFRRaPetkq2sECuFDQaVoIwowFtX3cJMTWQeD/8sL0lFx3tPwC5sTuwo=)